testing/integration-tests/inferred-spans-test/build.gradle.kts

```diff
@@ -8,7 +8,7 @@ dependencies {
   testImplementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations")
 }
 
-tasks.test {
+tasks.withType<Test>() {
   jvmArgs(
     //"-Dotel.javaagent.debug=true",
     "-Delastic.otel.inferred.spans.enabled=true",
```

Comment:
Did you test that this still works with the property on/off?

Reply:
What do you mean with property on off?
In case you are targeting at whether the tests actually test what they should: yes they do. This test actually uncovered in a previous iteration that the extension wasn't wokring correctly because the service files wouldn't get merged.